### DISCUSSION TITLE
Create a md note with flow launcher

### DISCUSSION THREAD

**Discussion by Warden20** [Flow-Launcher/Flow.Launcher#508] [Ideas]
I handle a lot of md notes and it is a problem to be going to the specific application and creating it.
I would like to know if you could implement something similar to this https://github.com/mrjackphil/qnote-ahk
It is a solution but if it were integrated into flow launcher like a puglin or a feature it would be the best solution
For example, if I open Flow Launcher and with the word note, what follows would be the note and that has the name 2020-11-05, the day of creation would be excellent
Would it be possible to create something like that?

**Comment by Zeroto521** [comment]
If you write a note in another place (not in the flow), then copy the note to flow.
That would be fine.

If you key in the flow one word by one word, that would be impossible.
The flow will record as "This sentence." as 14 files. And each file content as follows.

1. `T`
2. `Th`
3. `...`
13. `This sentence`
14. `This sentence.`

So you know the answer.

**Comment by taooceros** [comment]
I think it is possible to register another hotkey to save the file instead of saving it every time querying. For example, WoxDictionary do register `alt+enter` for copy query. Or maybe it is possible to create a window with a plugin action keyword similar to the one in quote.
However, I don't think Flow has the access to the current working directory, so maybe only one customized location is allowed.

**Comment by jjw24** [comment]
I wonder maybe something like a plugin where you could query `note:` hit enter and you can get a pop up screen like in the example and the screen will do exactly that is showing, create file and append the text in.

or alternatively, set the text file in the plugin and `query note: file: blah.md text: some text`. So Flow will know to pass to the plugin on `note:` then the plugin will look for `file:` and find blah.md in settings for the location, then look for `text:` and append the text that follows to the file.

Although, both solutions will require user to indicate some sort of finished query by hitting enter on the result.

Edit: on second thought I think first solution is much doable and allows you to amend the already entered text, as in the new window that will be opened will actually open the contents of the file

**Comment by maholli** [comment]
+1 for this

**Comment by Warden20** [comment]
An alternative could be to use a specific Inbox.md in a custom destination. And trigger the entry with a query in flow launcher like "j" and then write the content of the note that will go inside Inbox.md
